checkov/arm/context_parser.py

```diff
@@ -43,10 +43,9 @@ def evaluate_default_parameters(self):
         for key_entry in keys_w_params:
             param = re.sub("\[parameters\('|'\)\]", "", self._get_from_dict(dict(self.arm_template),
                                                                             key_entry[:-1])[key_entry[-1]])
-            self._set_in_dict(dict(self.arm_template), key_entry, parameter_defaults[param])
-            logging.debug(
-                "Replacing parameter {} in file {} with default value: {}".format(param, self.arm_file,
-                                                                                  parameter_defaults[param]))
+            if param in parameter_defaults:
```

Comment:
@schosterbarak Handled case of parameter having `null` as the default value, which means it does not have a default value